src/baffle.js

```diff
@@ -8,7 +8,8 @@ import Obfuscator from './obfuscator';
 
 const defaults = {
     characters: 'AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz~!@#$%^&*()-+=[]{}|;:,./<>?'.split(''),
-    speed: 50
+    speed: 50,
+    exclude: [' '],
```

Comment:
Could we drop the trailing comma? And maybe reorder to keep speed last (alphabetical) while we're at it?